client/landing/jetpack-cloud/components/sidebar/index.jsx

```diff
@@ -168,7 +173,9 @@ class JetpackCloudSidebar extends Component {
 							materialIcon="settings"
 							materialIconStyle="filled"
 							selected={ this.isSelected( '/settings' ) }
-						/>
+						>
+							{ settingsIssues.length > 0 && <Badge type="error">{ settingsIssues.length }</Badge> }
```

Comment:
instead of using the badge here can we use the new `showNotificationDot` prop in the SidebarItem. See https://github.com/Automattic/wp-calypso/pull/40259